chatbot.py

```diff
@@ -0,0 +1,62 @@
+from chatterbot import ChatBot
+from chatterbot.trainers import ChatterBotCorpusTrainer
+import speech_recognition as sr
+from gtts import gTTS
+import os
+import logging
+
+logger = logging.getLogger()
+logger.setLevel(logging.CRITICAL)
+
+class MyChatBot:
+    
+    def __init__(self, **kwargs):
+        self.chatbot = ChatBot('Robot')
+        #chatbot için yeni bir eğitim oluşturun
+        self.trainer = ChatterBotCorpusTrainer(self.chatbot)
+
+        self.trainer.train("chatterbot.corpus.turkish")
+        # Chatbot'u Türkçe bütüncesine"(corpusuna)"göre eğitin
+        self.trainer.train("chatterbot.corpus.turkish.greetings")
+        # Chatbot'u Türkçe Selamlama bütüncesine"(corpusuna)"göre eğitin
+        self.trainer.train("chatterbot.corpus.turkish.conversations")
+        # Chatbot'u Türkçe Konuşma bütüncesine"(corpusuna)"göre eğitin
+
+        self.recognizer = sr.Recognizer()
+
+    def get_audio_input(self):
```

Comment:
The module https://github.com/dmt-labs/modular-biped/blob/develop/modules/speechinput.py already handles speech input, so there is no need to re-create that funcitonality here. This class can just subscribe to the 'speech' event and process the `msg` being sent there.